java/RSPEC-128.java

```diff
@@ -0,0 +1,14 @@
+class SwitchNoBreak {
```

Comment:
RSPEC-2176: Class has same name as parent
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._